Tensile/Source/lib/source/ContractionProblem.cpp

```diff
@@ -485,12 +485,19 @@ namespace Tensile
         m_batchSizes.resize(m_batchIndices.size());
         m_boundSizes.resize(m_boundIndices.size());
 
+        m_freeIndicesA.resize(m_a.dimensions(), -1);
```

Comment:
a.dimensions() includes all dims of a - free, batched,and bound, so this will 'oversize' a vector meant to contain just free indices for A.   Make sure we document this may contain -1/invalid values.

Reply:
Yeah, the oversized vector (and -1's) lets us recreate the order IndexAssignmentsA in the c++ environment. Just added some comments to the header to help explain that freeIndicesA contains extra values of -1.